cdc/processor/pipeline/sink.go

```diff
@@ -194,6 +192,7 @@ func (n *sinkNode) addRowToBuffer(ctx context.Context, event *model.PolymorphicE
 		return nil
 	}
 
```

Comment:
I think it is better to remove  `addRowToBuffer` , since there is no buffer in `SinkNode` anymore. The `emitRowToSink` is enough.

Reply:
Addressed.